README.md

```diff
@@ -140,6 +140,87 @@ See the issue and its discussion [here](https://github.com/PyCQA/pylint/issues/1
+
+Now decompress the compressed image with: `unxz ubuntu-16.04.2-mate-odroid-xu4-20170510.img.xz` to obtain the img file.
+
+Use the command `lsblk` to identify the block device that is the eMMC/microSD card you are trying to flash.
+To be sure, unplug your adapter first and run `lsblk`, then plug in the adapter and run `lsblk` to see the difference.
+In my case it was `/dev/sdb`.
+
+Finally, to flash the image onto the card, you can use the `ddrescue` unix library.
+
+To install it run `sudo apt-get install ddrescue`.
+
+To flash it run `sudo ddrescue -D --force ubuntu-16.04.2-mate-odroid-xu4-20170510.img.xz /dev/sdb`, making sure to replace `/dev/sdb` with your own proper value.
+
+By the time it is done, you will have a fresh install of ubuntuMATE on your card.
+**Note** that when booting up for the very first time after a flash the odroid will boot to login screen, but then suddenly die.
+This is normal,  after the initial bootup you will need to reboot one more time for it to finally work properly.
+
+Note that the first time you connect your odroid to the internet it will most likely automatically start the job `apt-daily.service` which will temporarily make the `apt-get` command unavailable.
+To confirm this, you can check with `ps aux | grep apt` to get the process id (pid). This process will lock the use of `apt-get`.
+The best thing to do is to just wait until it is done. You can see how long it has been running with `ps -o etime= -p process_id` (replacing "processid" with the corresponding pid of the apt daily update process.
+To get a running time of elapsed time updated each second you may run `watch -n 1 ps -o etime= -p process_id`.
+
+### Custom images
+
+These custom images are made by directly setting up the changes on a fresh image, then using `dd` to save the state of the operating system as an image file.
+
+Example: `sudo dd if=/dev/sdb of=ubuntuMateDev.img bs=4M`
+
+For step by step instructions see [here](http://odroid.us/mediawiki/index.php?title=Create_SD_Card_Images).
+
+#### Dev
+
+The ubuntuMATE image is customized as follows to allow us to more easily develop on the odroid.
+
+Differences between default ubuntuMATE image:
+- [Autologin enabled](https://ubuntu-mate.community/t/auto-login-to-the-desktop/60)
+- Install `curl` and `git` with `apt-get`
+- Clone this `robotics-prototype` repo to `~/Programming` folder
+- Install `nvm v0.33.11`, `node v10.13.0` (LTS) by `wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash` , and `nvm install node v10.13.0`, respectively
+- Setup systemd service called `ip-emailer` to run `runEmailer.sh` ([how to setup a startup service with systemd](https://gist.github.com/pdp7/d4770a6ba17e666848796bf5cfd0caee))
+- Add configuration files `.bash_aliases` and `.nanorc`
+
+Any kind of authentication requiring services (such as a connection to a wifi network requiring username/password) will not be included in the images.
```

Comment:
the services won't be included, or their configurations won't be included ?

Reply:
both, I meant like we won't add anything to these images that requires authentication because most of the time it is stored in a file somewhere which anyone could read if they know the odroids sudo password. For example, not adding my wifi credentials for concordia university. Perhaps you thought I meant services as in systemd but I meant it more generally speaking.

Reply:
so you will still be able to connect to WIFI with this image, it just won't have any saved WIFI configuration data if it relates to authentication